readme.md

```diff
@@ -643,6 +643,7 @@
 - [Password Cracking](https://github.com/n0kovo/awesome-password-cracking#readme) - The process of recovering passwords from data that has been stored in or transmitted by a system in scrambled form.
 - [Security Card Games](https://github.com/Karneades/awesome-security-card-games#readme) - Train your skills and discuss various security topics.
 - [Suricata](https://github.com/satta/awesome-suricata#readme) - Intrusion detection/prevention system and network security monitoring engine.
+- [Microsoft Sentinel](https://github.com/kfriede/awesome-microsoft-sentinel#readme) - Microsoft Sentinel cloud-native SIEM/SOAR solution.
```

Comment:
The description should not contain the list title. Also not clear what SIEM/SOAR is. Needs a better description in general.